packages/web-app-files/src/mixins/actions/navigate.ts

```diff
@@ -75,7 +75,7 @@ export default {
       }
 
       if (isLocationSpacesActive(this.$router, 'files-spaces-project')) {
-        return createLocationPublic('files-spaces-project')
+        return createLocationSpaces('files-spaces-project')
```

Comment:
Probably worth a bugfix changelog item.

Reply:
Would have done that if I had understood what the code is good for 😂 
just fixed it based on the type...

Reply:
```typescript
export const createLocationPublic = (name: shareTypes, location = {}): Location =>
  createLocation(name, location)
  
export const createLocationSpaces = (name: spaceTypes, location = {}): Location =>
  createLocation(name, location)
``` 
I think it's just a code fix without any implication for the user. But at least investigating this made me find the "open folder" issue...